ManatiCR/retopais
# RP-38: Footer mesa logos

How-to-test:
- Go to http://retopais.dev/node/add/alliance
  - Type field should be multivalue and should have a new Mesa option
  - Create a node with Mesa de Innovación type selected
  - Go to http://retopais.dev/admin/structure/views/view/alianzas/edit/panel_pane_4, you should get that node in the preview (without link).
